doc/sphinx/source/recipes/recipe_zec.rst

```diff
@@ -0,0 +1,110 @@
+.. _recipes_zec:
+
+Zero Emissions Commitment (ZEC)
+===============================
+
+Overview
+--------
+
+The Zero Emissions Commitment (ZEC) quantifies the change in global mean
+temperature expected to occur after net carbon dioxide (CO\ :sub:`2`)
+emissions cease. ZEC is therefore important to consider when estimating
+the remaining carbon budget. Calculation of ZEC requires dedicated simulations
+with emissions set to zero, branching off a base simulation with emissions.
+In CMIP6 the simulations were part of ZECMIP, with the simulations called
+``esm-1pct-brch-xPgC`` branching off the ``1pctCO2`` simulation when emissions
+reach x PgC. The default x was 1000PgC, with additional simulations for 750PgC
+and 2000PgC. In CMIP7, ZEC simulations (``esm-flat10-zec``) are part of the
+fast track and branch off (``esm-flat10``) with constant emissions of 10GtC/yr
+at year 100 (Sanderson 2024).
+
+
+Available recipes and diagnostics
+---------------------------------
+
+Recipes are stored in recipes/
+
+   * recipe_zec.yml
+
+
+Diagnostics are stored in diag_scripts/
+
+   * :ref:`climate_metrics/zec.py <api.esmvaltool.diag_scripts.climate_metrics.zec>`
+
+
+User settings in recipe
+-----------------------
+
+* Preprocessor
+
+   * ``area_statistics`` (*operation: mean*): Calculate global mean.
+   * ``annual_statistics`` (*operation: mean*): For the ZEC simulation,
+     compute annual statistics.
+   * ``climate_statistics`` (*operation: mean, period: full*): For the
+     reference simulation, compute the 20-year time average around the
+     time when the ZEC simulation starts.
```

Comment:
Are users of the recipe expected to change these settings and is the result still correct then?

Reply:
Nope, they should be kept. Where should I best make it clear?

Reply:
You can just remove them from the documentation page, so we only document those settings that can be changed by users of the recipe. I think that's what we've done so far. Or, if you prefer to keep them, you could add a note that these should not be changed.